src/ChainSafe.Gaming.HyperPlay/HyperPlayRequest.cs

```diff
@@ -0,0 +1,40 @@
+using Newtonsoft.Json;
+
+namespace ChainSafe.Gaming.HyperPlay
+{
+    /// <summary>
+    /// Request model for HyperPlay RPC requests.
+    /// </summary>
+    public struct HyperPlayRequest
+    {
+        [JsonProperty("request")]
+        public Request Request { get; set; }
+
+        [JsonProperty("chain")]
+        public Chain Chain { get; set; }
+    }
+
+    public struct Request
+    {
+        [JsonProperty("method")]
+        public string Method { get; set; }
+
+        [JsonProperty("params", DefaultValueHandling = DefaultValueHandling.Ignore)]
+        public object[] Params { get; set; }
+    }
+
+    public struct Chain
+    {
+        [JsonProperty("chainId")]
+        public string ChainId { get; set; }
+    }
+
+    /// <summary>
+    /// Error model for HyperPlay RPC responses.
+    /// </summary>
+    public struct HyperPlayError
```

Comment:
The file name doesn't really reflect the contents IMO. I would rather break this up into separate files under the "Dto" folder or rename this file to something like HyperPlayDtos.

Reply:
gotcha, will change